tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -16,6 +16,13 @@
     "password": None,
     {% endif %}
     "db": "{{ MONGODB_DATABASE }}",
+    {% if MONGODB_REPLICA_SET %}
+    "replicaSet": "{{ MONGODB_REPLICA_SET }}",
+    {% endif %}
+    "ssl": {% if MONGODB_USE_SSL %} True {% else %} False {% endif %}
```

Comment:
We need to add a coma "," at the end of the line, otherwise it will cause a parsing error when "authsource" is defined.

Reply:
I have fixed this as suggested